src/mappings/core.ts

```diff
@@ -446,9 +458,18 @@ export function handleSwap(event: SwapEvent): void {
   token1HourData.untrackedVolumeUSD = token1HourData.untrackedVolumeUSD.plus(amountTotalUSDTracked)
   token1HourData.feesUSD = token1HourData.feesUSD.plus(feesUSD)
 
+  token1BlockData.volume = token1BlockData.volume.plus(amount1Abs)
+  token1BlockData.volumeUSD = token1BlockData.volumeUSD.plus(amountTotalUSDTracked)
+  token1BlockData.untrackedVolumeUSD = token1BlockData.untrackedVolumeUSD.plus(amountTotalUSDTracked)
+  token1BlockData.feesUSD = token1BlockData.feesUSD.plus(feesUSD)
+
   swap.save()
   token0DayData.save()
+  token0HourData.save()
```

Comment:
Added a save here for hourly data. It looks like this isn't being saved right now.